pymyq/api.py

```diff
@@ -399,12 +401,14 @@ async def _oauth_authenticate(self) -> Tuple[str, int]:
                     "Unable to continue login process."
                 )
 
+            data.update(OAUTH_LOGIN_ADDITIONAL_DATA)
+
             # Perform login to MyQ
             _LOGGER.debug("Performing login to MyQ")
             resp, _ = await self.request(
                 method="post",
                 returns="response",
-                url=resp.url,
+                url=OAUTH_LOGIN_URI,
```

Comment:
I believe resp.url was typical /LoginWithemail

Any idea what was different between it and this?

Reply:
It was returning `/Login`, but the submit action on the web form is `/LoginWithEmail`

Reply:
I actually didn't dump that to sanity check it. The comments in the other thread made it sound like this had changed and hardcoding was necessary, but maybe I misunderstood. I'll test the original URL once my rate limit expires in about 20 minutes.

Reply:
> It was returning `/Login`, but the submit action on the web form is `/LoginWithEmail`

Interesting - I could have sworn I saw LoginWithEmail in the past - but perhaps that change was part of their most recent changes. Thanks

@justinlindh when you have a chance to test and confirm everything - undraft and I will merge

Reply:
Yeah I'm assuming they updated the header on their side for whatever reason to be different from the actual submit path in the form. Ideally we'd extract the submit path from the login form dynamically rather than relying on the response headers or a const. When I called it out in the other thread it was just to point out the discrepancy, so this is a bit of a quick-and-dirty fix, which is probably fine for now, but may wish to be more robust when someone has time to do so.

Reply:
I get redirected to `/LoginWithEmail` from the initial /connect/authorize call, not `/Login`.

Reply:
/shrug Not sure, just what I observed when testing. Who knows with MyQ, they seem to be in constant flux. Either way, it makes more sense to get the form post URL from the form itself than from response headers imo.

Reply:
Looks like it's returning this for `resp.url'

```
https://partner-identity.myq-cloud.com/Account/LoginWithEmail?returnUrl=/connect/authorize/callback?client_id%3DANDROID_CGI_MYQ%26code_challenge%3<REDACTED>%26code_challenge_method%3DS256%26redirect_uri%3Dcom.myqops%253A%252F%252Fandroid%26response_type%3Dcode%26scope%3DMyQ_Residential%2520offline_access
```

So it actually is the `LoginWithEmail` path.

Tested reverting back to using that value and the plugin still succeeds. Will push an update to remove that part now.

Reply:
Ah yes, it looks like this was a difference in packages, not responses. The (Go) package I was debugging from was referencing the incorrect URL. Apologies for the confusion.